session_test.go

```diff
@@ -2351,6 +2351,17 @@ func (s *testSessionSuite) TestSpecifyIndexPrefixLength(c *C) {
 	se.Close()
 }
 
+func (s *testSessionSuite) TestColumnLength(c *C) {
```

Comment:
we can put this test in `executor/write_test.go: TestInsert`,
and comment which issue is this case for.